packages/nimble-components/src/anchor/template.ts

```diff
@@ -43,19 +41,26 @@ AnchorOptions
         aria-relevant="${x => x.ariaRelevant}"
         aria-roledescription="${x => x.ariaRoledescription}"
         ${ref('control')}
+    >${
+    /* Start and End slot templates inlined to avoid extra whitespace.
+        See https://github.com/microsoft/fast/issues/6557 */ ''
+}${
+    /* Whitespace intentionally avoided between tags for inline styles */ ''
+}<span
```

Comment:
Can you merge these two comments together? no compelling reason to have them separate back to back